code/__DEFINES/reagents.dm

```diff
@@ -30,6 +30,8 @@
 #define CLEAR_REAGENTS	4	// all reagents were cleared
 
 #define MIMEDRINK_SILENCE_DURATION 30  //ends up being 60 seconds given 1 tick every 2 seconds
+#define TRESHOLD_UNHUSK 50 //Health treshold for synthflesh and rezadone to unhusk someone
```

Comment:
treshold? do you mean threshold?

Reply:
Ah yes, I've corrected that four times it was misspelled now